series/supportedseries_test.go

```diff
@@ -22,6 +24,13 @@ func (s *supportedSeriesSuite) SetUpTest(c *gc.C) {
 	s.CleanupSuite.SetUpTest(c)
 	cleanup := series.SetSeriesVersions(make(map[string]string))
 	s.AddCleanup(func(*gc.C) { cleanup() })
+
+	series.RestoreUbuntuSeries()
+	s.AddCleanup(func(*gc.C) { series.RestoreUbuntuSeries() })
```

Comment:
is this intended? don't we just want it in the cleanup?

Reply:
Not intended, thanks.